webpack.test.js

```diff
@@ -0,0 +1,137 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const angularCommon = path.resolve(__dirname, 'node_modules', 'esn-frontend-common-libs', 'src', 'angular-common.js');
+const angularInjections = path.resolve(__dirname, 'src', 'require-angular-injections.js');
+const chartJs = path.resolve(__dirname, 'node_modules', 'esn-frontend-common-libs', 'src', 'frontend', 'components', 'Chart.js/Chart.js')
+const materialAdmin = path.resolve(__dirname, 'node_modules', 'esn-frontend-common-libs', 'src', 'frontend', 'js', 'material.js');
+const momentPath = path.resolve(__dirname, 'node_modules', 'moment', 'moment.js');
+const chaiPath = path.resolve(__dirname, 'node_modules', 'chai/chai.js');
+const pugLoaderOptions = {
+  root: `${__dirname}/node_modules/esn-frontend-common-libs/src/frontend/views`
+};
+const lodashPath = path.resolve(__dirname, 'node_modules', 'lodash', 'dist', 'lodash.js');
+const i18nLoaderMockPath = path.resolve(__dirname, 'test', 'config', 'mocks', 'i18n-loader.js');
+const icalPath = path.resolve(__dirname, 'node_modules', 'esn-frontend-common-libs', 'src', 'frontend', 'components', 'ical.js', 'build', 'ical.min.js');
+
+const BASE_HREF = process.env.BASE_HREF || '/';
+
+module.exports = {
```

Comment:
Try using `webpack-merge`.

Reply:
But the webpack config for testing is different with the common one. Like, a lot of different points.